### DISCUSSION TITLE
三维标绘无法编辑

### DISCUSSION THREAD

**Discussion by Dennis9937** [dengxiaoning/cesium_dev_kit#69] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/6850d734-96f3-4e26-a010-fe9fe794b10f)

三维标绘里面无法编辑呢，像这种钳击、攻击箭头希望提供编辑操作 :bulb:

**Comment by dengxiaoning** [comment, marked as answer]
Thank you for your donation #71 👍
